.vscode/settings.json

```diff
@@ -12,7 +12,7 @@
     "editor.defaultFormatter": "vscode.html-language-features"
   },
   "[javascript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "HookyQR.beautify"
```

Comment:
It shouldn't be changed. As a project setting, we should use "esbenp.prettier-vscode"